src/js/legacy/menu.js

```diff
@@ -1,28 +1,32 @@
-$(document).ready(function() {
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
jQuery appears to use `DOMContentLoaded` and fallback to `document.readyState === 'complete'` for older versions of IE. I think we're okay for now, and probably okay if we load bundles with `defer`.